EidosScribe/EidosHelpController.mm

```diff
@@ -158,6 +158,10 @@ - (void)dealloc
 // now substitutes an EidosNSString for the original NSString.  That EidosNSString responds to copyWithZone: by making a copy
 // of itself, preventing NSTaggedPointerString from getting in.  This seems to work, and I think it ought to be fairly safe
 // and airtight.  We shall see.  BCH 2/26/2018
+// BCH 11/27/2019: Note that the Qt version of this class now has a much nicer design, in which a proper tree class is used
```

Comment:
These are changes that occurred outside of your work, and are already committed to the openmp branch, so there seems to be a problem with the rebasing of this PR, or something.  I have just released SLiM 3.3.2, and just pulled over those changes into the openmp branch, so you ought to rebase onto what is not the head of the openmp branch for cleanliness.  Sorry things keep shifting; life working on a branch.